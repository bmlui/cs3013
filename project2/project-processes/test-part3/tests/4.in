/usr/bin/false && echo This should not print 
exit

