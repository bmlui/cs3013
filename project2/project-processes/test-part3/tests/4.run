./wshell < tests/4.in
