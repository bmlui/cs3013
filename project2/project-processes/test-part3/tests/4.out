test-part3$ /usr/bin/false && echo This should not print 
test-part3$ exit
